### ISSUE TITLE
Attach labels (tags) to logs

### ISSUE THREAD

**Issue by svetlanabrennan** [newrelic/node-newrelic#2716]
### Description

We need to support adding labels (tags) to logs.

TLDR:
- Labels disabled by default
- Labels should be prefixed with "tags."
- Labels are added to common attributes, not individual logs.
- Allows individual labels to be excluded

Dependent on this issue that adds configurations to the agents to enable this feature: https://github.com/newrelic/node-newrelic/issues/2713

### Implementation details:
- If `application_logging.forwarding.labels.enabled` is set to `true`, all labels **MUST** be attached in `common` attributes unless there is matching entry in `application_logging.forwarding.labels.exclude`.
- The attribute key **MUST** be the prefix `tags.` followed by the `label name`.
- Except for `tags.`, the attribute **MUST** match the case of the original label. Example, `MYLABEL;Value1` becomes `"tags.MYLABEL": "Value1"`.
- Add supportability metric for labels - `Supportability/Logging/Labels/{language}/{enabled\|disabled}`. **MUST** be generated by agents when labels on application logging is enabled. `disabled` **SHOULD** be generated if possible, although some agents' implementation may preclude it.  It is expected to be sent once per agent connect (not per harvest).|

Example newrelic.config bit from .NET showing labels with no exclude configuration:
```
<labels>label1:value1;LABEL2:value2;LABEL2-ALSO:value3</labels>
```

Example `log_event_data` payload based on above newrelic.config:
```
[{
   "common": {
     "attributes": {
       "entity.guid": <ENTITY_GUID>,
       "entity.name": "My Application",
       "hostname": "my-host-name",
       "tags.label1": "value1",
       "tags.LABEL2": "value2",
       "tags.LABEL2-ALSO":"value3"
     }
   },
   "logs": [{
       "timestamp": <TIMESTAMP_IN_UNIX_EPOCH_MILLSECONDS>,
       "level": "INFO"
       "message": "User 'xyz' logged in",
       "span.id": "123456789ADF",
       "trace.id": "ADF09876565"
     }]
}]
```

Same example above, but with the following exclude configuration (.NET agent XML):
```
<applicationLogging>
  <forwarding>
    <labels enabled="true" exclude="label2" />
  </forwarding>
</applicationLogging>
```

Example `log_event_data` payload based on above newrelic.config - no longer has LABEL2, but still has LABEL2-ALSO:
```
[{
   "common": {
     "attributes": {
       "entity.guid": <ENTITY_GUID>,
       "entity.name": "My Application",
       "hostname": "my-host-name",
       "tags.label1": "value1",
       "tags.LABEL2-ALSO":"value3"
     }
   },
   "logs": [{
       "timestamp": <TIMESTAMP_IN_UNIX_EPOCH_MILLSECONDS>,
       "level": "INFO"
       "message": "User 'xyz' logged in",
       "span.id": "123456789ADF",
       "trace.id": "ADF09876565"
     }]
}]
```
[Link to labels spec](https://source.datanerd.us/agents/agent-specs/blob/main/Labels.md#labels). 

[Discovery doc ](https://newrelic.atlassian.net/wiki/spaces/~712020468958465767472dae555831f9e3cc31/pages/3678240795/APM+Agent+Logging+Include+APM+Custom+Tags+in+Logs+For+Better+Insights)about labels.

**Comment by bizob2828** [comment]
docs pr https://github.com/newrelic/docs-website/pull/19319
